### ISSUE TITLE
bigint negative value fail

### ISSUE THREAD

**Issue by augustinyiptong** [JuliaDatabases/ODBC.jl#152]
I have a table with a bigint column in sql server 2012

`sql_query = """select bb from tt"""`
> bb
> 1	Nullable{Int64}(1)
> 2	Nullable{Int64}(3)
> 3	Nullable{Int64}(71047936)
> 4	Nullable{Int64}(5)
> 

I can do:
`sql_query = """select cast(bb as int) as bb from tt"""`

> 	bb
> 1	Nullable{Int32}(1)
> 2	Nullable{Int32}(3)
> 3	Nullable{Int32}(-5)
> 4	Nullable{Int32}(5)

The 3rd row is correctly returned as -5

**Comment by quinnj** [comment]
There were updates just merged to master; could you try the original example again with the latest code? Running `Pkg.checkout("DataFrames"); Pkg.checkout("ODBC")` should get you the latest code.

**Comment by abx78** [comment]
I am experiencing some issues with the MSSQL driver. I originally found it on BIGINT data type, however I think it might be happening also on other types.
Here the offending code:
```
using ODBC
connection_string = "Driver={SQL Server};Server=localhost;Database=ReSolver_local_dev;Trusted_Connection=yes"
connection = ODBC.DSN(connection_string, prompt=false)
stmt = "SELECT CAST(bigint AS BIGINT) bigint, 
CAST(int AS int) int, 
CAST(smallint AS smallint) smallint, 
CAST(numeric AS numeric) numeric, 
CAST(float AS float) float, 
CAST(real AS real) real 
FROM  
(
VALUES 
    (-1.0, -1.0, -1.0, -1.0, -1.0, -1.0),  
    (null, null, null, null, null, null),  
    (1.0, 1.0, 1.0, 1.0, 1.0, 1.0)
) V 
(bigint, int, smallint, numeric, float, real)"
ODBC.query(connection, stmt)
```

The error:

> [ODBC] 22003: [Microsoft][ODBC SQL Server Driver]Numeric value out of range
> ERROR: ODBC.ODBCError("API.SQLFetchScroll(q.stmt, API.SQL_FETCH_NEXT, 0) failed; return code: -1 => SQL_ERROR")
> Stacktrace:
>  [1] ODBC.Query(::ODBC.DSN, ::String) at C:\Program Files\ReSolver.DistributedJulia\depot\packages\ODBC\YEzHX\src\ODBC.jl:57
>  [2] #query#15(::Bool, ::Bool, ::Dict{Int64,Function}, ::Function, ::ODBC.DSN, ::String, ::Type{DataFrame}) at C:\Program Files\ReSolver.DistributedJulia\depot\packages\ODBC\YEzHX\src\Query.jl:390
>  [3] query at C:\Program Files\ReSolver.DistributedJulia\depot\packages\ODBC\YEzHX\src\Query.jl:385 [inlined]
>  [4] query(::ODBC.DSN, ::String) at C:\Program Files\ReSolver.DistributedJulia\depot\packages\ODBC\YEzHX\src\Query.jl:376
>  [5] top-level scope at none:0
 
As far as I know, data types in SQL are not unsigned so all the above examples should work correctly.

**Comment by quinnj** [comment]
This will be fixed in #264
